src/keymap.cpp

```diff
@@ -803,8 +798,11 @@ std::vector<MacroCmd> KeyMap::get_macro(const std::string& key)
 
 bool KeyMap::is_valid_context(const std::string& context)
 {
-	for (unsigned int i = 0; contexts[i] != nullptr; i++) {
-		if (context == contexts[i]) {
+	if (context == "all") {
+		return true;
+	}
+	for (const auto& ctx : contexts) {
```

Comment:
Can this loop be replaced by `std::map::find()`?

Reply:
Yes, nice improvement, done (using `std::map::count()` instead because I find it slightly easier to read than having to compare agains `std::map::end()`).